tools/kmodules/lisa-prerequisites/android/aosp/0001-fs-Move-filp_open-kernel_read-write-outside-the-name.patch

```diff
@@ -0,0 +1,53 @@
+From 18197865626ae79292543f9de0db9d9cb42b7902 Mon Sep 17 00:00:00 2001
+From: Kajetan Puchalski <kajetan.puchalski@arm.com>
+Date: Wed, 7 Jun 2023 11:08:08 +0000
+Subject: [PATCH] fs: Move filp_open, kernel_read & write outside the namespace
+
+filp_open, kernel_read & kernel_write are symbols required by the Lisa
+module's pixel6_emeter feature. By default they are moved into a
+restricted namespace and disabled on Android product kernels. In order
+for the module to work, move the symbols back into the global namespace.
+
+---
+ fs/open.c       | 2 +-
+ fs/read_write.c | 4 ++--
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/fs/open.c b/fs/open.c
+index b9d55ca3763b..d29008121b8a 100644
+--- a/fs/open.c
++++ b/fs/open.c
+@@ -1158,7 +1158,7 @@ struct file *filp_open(const char *filename, int flags, umode_t mode)
+ 	}
+ 	return file;
+ }
+-EXPORT_SYMBOL_NS(filp_open, ANDROID_GKI_VFS_EXPORT_ONLY);
```

Comment:
I think we can avoid touching exports just to remove ANDROID_GKI_VFS_EXPORT_ONLY namespace, we already import the namespace (VFS_internal_I_am_really_a_filesystem_and_am_NOT_a_driver) in the module source. Reducing this patch will reduce the chances of conflict when applying it.

EDIT: if that works that actually allows removing the entire patch, so even better.